WooCommerce/Classes/ViewRelated/ReusableViews/ActivitySpinnerAndLabelTableViewCell.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+class ActivitySpinnerAndLabelTableViewCell: UITableViewCell {
```

Comment:
I know this is still WIP, but this could be final.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4910/commits/8c373612c6e33caf4fd8227777a42d7959f0d41e